src/apps/main/signup/signup.js

```diff
@@ -11,7 +11,6 @@ export default function SignUp ({ formProps }) {
         <Form {...formProps}>
           <TextInput label='Email Address' name='email' placeholder='Your Email' required isFormField />
           <TextInput type='password' label='Password Address' name='password' placeholder='Your Password' required isFormField />
-          <ErrorOrHint name='password' isFormField />
```

Comment:
Yup, looks like the inlcuded TextInput component already renders a `ErrorOrHint` component.